spanner/spansql/sql_test.go

```diff
@@ -190,6 +190,22 @@ func TestSQL(t *testing.T) {
 			"ALTER TABLE Ta SET ON DELETE CASCADE",
 			reparseDDL,
 		},
+		{
+			&AlterTable{
```

Comment:
Should we add a test for `SetColumnType` as well?

Reply:
Sure, done.